client/app/controllers/collection/index.js

```diff
@@ -93,6 +94,9 @@ export default Ember.Controller.extend({
         },
         changeRoute(path){
             this.transitionToRoute(path);
+        },
+        changeView(viewName) {
+            this.set('view', viewName);
```

Comment:
If we are not going to increase the types of views, you could reduce this to one variable and toggleProperty instead. Essentially ```cardView = true``` is cards, ```false``` is table.